### PR TITLE
[IMP] website: add new `s_sidegrid` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
This commit introduces the new `s_sidegrid` intro snippet.

Part of task-4077427
task-4077589

requires: https://github.com/odoo/design-themes/pull/853

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by stefanorigano — CHANGES_REQUESTED** [review]
All good @anso-odoo , neat 👍 

I'd just propose to slightly review the layout in order to make the text breath a bit more.


| this PR | proposal |
|--------|--------|
| image | image | 

changes applied:
- "Picture with 3" `g-col-lg-3 col-lg-3`  `grid-area: 1 / 5 / 10 / 8`;
- "Text Block" `g-col-lg-5 col-lg-5`  `grid-area: 1 / 8 / 10 / 13; --grid-item-padding-x: 26px;`

Thanks!

**Comment by anso-odoo** [comment]
@stefanorigano  indeed, done 👍

**Review by stefanorigano — APPROVED** [review]
LGTM, thanks!
~Let's finalise the design-theme pr~

edit: All good to me 👍

File: addons/website/views/snippets/s_sidegrid.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                <div class="o_grid_item o_grid_item_image g-height-4 g-col-lg-4 col-lg-2" style="z-index: 5; grid-area: 1 / 1 / 5 / 5;">
+                    <img class="img img-fluid rounded" src="/web/image/website.s_sidegrid_default_image_1" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image o_snippet_mobile_invisible g-height-9 g-col-lg-3 col-lg-2 d-lg-block d-none" style="z-index: 2; grid-area: 1 / 5 / 10 / 8;">
+                    <img class="img img-fluid rounded" src="/web/image/website.s_sidegrid_default_image_2" alt=""/>
+                </div>
+                <div class="o_grid_item o_grid_item_image o_snippet_mobile_invisible g-height-9 g-col-lg-4 col-lg-2 d-lg-block d-none" style="z-index: 4; grid-area: 5 / 1 / 14 / 5;">
+                    <img class="img img-fluid rounded" src="/web/image/website.s_sidegrid_default_image_4" alt=""/>
+                </div>
+                <div class="testanso o_grid_item o_grid_item_image o_snippet_mobile_invisible g-height-4 g-col-lg-8 col-lg-3 d-lg-block d-none" style="z-index: 3; grid-area: 10 / 5 / 14 / 13;">
```
Comment (SergeBayet) [inline addons/website/views/snippets/s_sidegrid.xml:21]: @anso-odoo probably a leftover? "testanso" :)

**Review by SergeBayet — CHANGES_REQUESTED** [review]
LGTM, just a little unnecessary class to remove :) see below.

File: addons/website/views/snippets/s_sidegrid.xml (hunk 1 above)
Comment (anso-odoo) [reply addons/website/views/snippets/s_sidegrid.xml]: Indeed, i'll remove that 👍

**Review by SergeBayet — APPROVED** [review]
LGTM. Thanks :)

**Comment by bso-odoo** [comment]
@robodoo delegate=SergeBayet

**Comment by SergeBayet** [comment]
@robodoo r+

**Pull request closed without merging**